scripts/lib/spreadsheet-mappings.ts

```diff
@@ -5,7 +5,7 @@ export type FlatAliasMap = { [index: string]: string };
 
 // Map from canonical column name to possible aliases in spreadsheets.
 export const FIELD_MAPPINGS: AliasMap = {
-  id: ['ID'],
+  id: ['ID', 'ID (StateAbv-number. eg., NY-1)'],
```

Comment:
I figured it was still useful to leave this guidance in the title, so I added it as a mapping